superset-frontend/src/explore/components/DisplayQueryButton.jsx

```diff
@@ -75,6 +75,24 @@ const CopyButton = styled(Button)`
   && {
     margin-left: ${({ theme }) => theme.gridUnit * 2}px;
   }
+
+  i {
+    padding: 0;
+  }
+`;
+
+const CopyButtonViewQuery = styled(Button)`
+  padding: ${({ theme }) => theme.gridUnit / 2}px
+    ${({ theme }) => theme.gridUnit * 2.5}px;
+  font-size: ${({ theme }) => theme.typography.sizes.s}px;
+
+  && {
+    margin-bottom: 5px;
```

Comment:
```suggestion
    margin-bottom: ${({ theme }) => theme.gridUnit}px;
```
I'm guessing can probably spare the extra pixel here just to snap things to the gridUnit ;)